docs/en/operations/settings/settings.md

```diff
@@ -460,6 +460,17 @@ Possible values:
 
 Default value: 1048576.
 
+## http_make_head_request {#http-make-head-request}
+
+Enables or disables execution of a HEAD request before the actual GET request
+
+Possible values:
+
+- 0 — Disabled.
+- 1 — Enabled.
+
+Default value: 1.
```

Comment:
```suggestion
## http_make_head_request {#http-make-head-request}

The `http_make_head_request` setting allows the execution of a `HEAD` request while reading data from HTTP to retrieve information about the file to be read, such as its size. Since it's enabled by default, it may be desirable to disable this setting in cases where the server does not support `HEAD` requests.

Default value: `true`.
```